### PR TITLE
Add a warning to the OpenETC client

### PR DESCRIPTION
**Pull request by hitchcott**
This PR will add a more visible warning to the client list to visually differentiate it from the more established clients.

Does this satisfy the concerns rasied so far?

Also, what do you think about including a refernce to the old version of Parity Ethereum / Open Etheruem client release that still supports ETC?

@cryptoasuka I am also cognescant that we don't want to make wording too negative. Perhaps we can agree on wording in this PR that satisfies all parties.

image

### CODE REVIEW

**Comment by hitchcott** [comment]
Replying to @TheEnthusiasticAs 

https://github.com/ethereumclassic/ethereumclassic.github.io/pull/529#issuecomment-664280882

> Yes, there still could be people who want to run one of those (OE, MG). A deprecation note was already put to those both on the website. Those can be taken down after a hard fork has been executed.

I'm am naively in favour of adding a links to deprecated clients for the historical record (including MG) with of of course and perhaps under a collapsed section to not confuse people. 

Perhaps only to their latest compatible release. e.g. https://github.com/openethereum/openethereum/releases/tag/v3.0.1

**Comment by TheEnthusiasticAs** [comment]
to this PR:

I find it repeated unnecessarily as there is already a note on the top of the website on that page. I would rephrase that top note to:

"The links on this page are contributions by ETC participants and are not guaranteed to be safe; use them at your own risk! Some of them could be experimental. Please do your own research before using any of resources on this site."

**Comment by TheEnthusiasticAs** [comment]
> Replying to @TheEnthusiasticAs
> 
> [#529 (comment)](https://github.com/ethereumclassic/ethereumclassic.github.io/pull/529#issuecomment-664280882)
> 
> > Yes, there still could be people who want to run one of those (OE, MG). A deprecation note was already put to those both on the website. Those can be taken down after a hard fork has been executed.
> 
> I'm am naively in favour of adding a links to deprecated clients for the historical record (including MG) with a warning of course and perhaps under a collapsed section to not confuse people.
> 
> Perhaps only to their latest compatible release. e.g. https://github.com/openethereum/openethereum/releases/tag/v3.0.1

Yes, I agree.
Maybe there could an "archived section" on the website someday.

**Comment by hitchcott** [comment]
I agree that the red warning is redundant, but as others have noted, the warning at the top is good for disclaiming purposes, but people will often just automatically skip reading those kind of banners. 

Having an additional warning next to the link is more likely to be read, as it pertains specifically to that link.

**Comment by hitchcott** [comment]
Another comment from @TheEnthusiasticAs 

https://github.com/ethereumclassic/ethereumclassic.github.io/pull/519#issuecomment-664315839

> My notes to this:
> Don't forget, based on the philosophy of this blockchain - open, permissionless, censorship-resistant - what the participants of this blockchain can do, is, to mark a contribution as "tested" or "untested".
> 
> There is a note on the client site on the website:
> "The links on this page are contributions by ETC participants and are not guaranteed to be safe; use them at your own risk! Please do your own research before using any of resources on this site."

I agree with that, too.

Getting the wording right on this PR is going to be an important precedent going forward.

That being said a client is a much more important piece of infrastructure than a link to an exchange or wallet, for example. We have some extra responsibility that we are not willy-nilly adding in promenant locations without proper warning.

---

(Btw, Please let's keep discussion in this thread to make sure we're all on the same page)

**Comment by TheEnthusiasticAs** [comment]
> I agree that the red warning is redundant, but as others have noted, the warning at the top is good for disclaiming purposes, but people will often just automatically skip reading those kind of banners.
> 
> Having an additional warning next to the link is more likely to be read, as it pertains specifically to that link.

I understand the concerns. My worry is, that the ETC participants enter the censoring field with saying: "Use other, because of a X client is experimental."
I am ok with an extra warning message, yes - because clients are an essential part of the infrastructure - similar to and not in red:

"This client is a new, experimental community project. Use it at your own risk."

File: content/development/clients/clients.en.yaml [hunk 1]
```diff
@@ -35,7 +35,8 @@ items:
     key: OpenETC
     name: OpenETC
     link: https://github.com/openetc
-    description: OpenETC is a fast and feature-rich multi-network Ethereum client. For Phoenix upgrade support, please use version 3.0.1 or later. Upgrade considerations, OpenETC is a branch of the deprecated ParityEthereum project. Users can use their existing data directories and do not need to resync for migration to OpenETC from the deprecated ParityEthereum v2.7.2 client. OpenETC is an open source, community maintained branch of the ParityEthereum client. Note this client project is marked as an experimental client due to its infancy. The open development team lacks a known track record of development due to their anonymous nature. Please do your own research if you intend to use this client.
+    warning: OpenETC is a new, experimental community project. For production deployments, please consider a more mature client.
```
Comment (TheEnthusiasticAs) [inline content/development/clients/clients.en.yaml:38]: please see my suggestion here.
https://github.com/ethereumclassic/ethereumclassic.github.io/pull/531#issuecomment-664374247
It should sound not censoring.

**Comment by TheEnthusiasticAs** [comment]
@gitr0n1n :-) We are on the same thought line. I mentioned it here above. Btw, would you please move your this comment https://github.com/ethereumclassic/ethereumclassic.github.io/pull/529#issuecomment-664448655 to here as the discussion should be continued here (see the ask by hitchcott https://github.com/ethereumclassic/ethereumclassic.github.io/pull/531#issuecomment-664346607). Thank you!

**Comment by gitr0n1n** [comment]
> Replying to @TheEnthusiasticAs
> 
> [#529 (comment)](https://github.com/ethereumclassic/ethereumclassic.github.io/pull/529#issuecomment-664280882)
> 
> > Yes, there still could be people who want to run one of those (OE, MG). A deprecation note was already put to those both on the website. Those can be taken down after a hard fork has been executed.
> 
> I'm am naively in favour of adding a links to deprecated clients for the historical record (including MG) with a warning of course and perhaps under a collapsed section to not confuse people.
> 
> Perhaps only to their latest compatible release. e.g. https://github.com/openethereum/openethereum/releases/tag/v3.0.1

Adding this comment to aggregate the convo to this PR:
https://github.com/ethereumclassic/ethereumclassic.github.io/pull/529#issuecomment-664448655

**Comment by hitchcott** [comment]
Ok @TheEnthusiasticAs , I replaced the "use a more mature client forproduction" bit with "For production deployments, please consider a more mature client."

If this is ok, I think we can merge it.
